README.md

```diff
@@ -1,3 +1,11 @@
+<p align="center"><a href="https://twill.io" target="_blank"><img src="http://cms-toolkit-dev.imgix.net/null47d1f04f-da9b-45f4-86cb-35a74e71c384/twill_logo.svg" width="400"></a></p>
```

Comment:
As a possible improvement, the image could be placed under the twill's domain: https://twill.io/logo.svg